incident/templates/incident/_lawsuit_rows.html

```diff
@@ -1,29 +1,32 @@
 {% load wagtailcore_tags %}
+{% load case_tags %}
 
-{% if page.lawsuit_name %}
+{% if page.case_number %}
 	<tr class="data-table__row">
 		<th class="data-table__label">
-			Name of Lawsuit
+			Case number
 		</th>
 		<td class="data-table__value">
-			<a href="{% pageurl page.get_parent %}?lawsuit_name={{ page.lawsuit_name }}">
-				{{ page.lawsuit_name }}
+			<a href="{% pageurl page.get_parent %}?case_number={{ page.case_number }}">
+				{{ page.case_number }}
 			</a>
 		</td>
 	</tr>
 {% endif %}
 
-{% if page.venue.all %}
+{% if page.case_statuses|length > 0 %}
 	<tr class="data-table__row">
 		<th class="data-table__label">
-			Venue
+			Case Status
 		</th>
 		<td class="data-table__value">
-			<ul class="data-table__list">
-				{% for venue in page.venue.all %}
-					<li class="data-table__list-item data-table__list-item--inline">
-						<a href="{% pageurl page.get_parent %}?venue={{ venue.pk }}">{{ venue }}</a></li>
-				{% endfor %}
+			{% for case_status in page.case_statuses %}
+			<a href="{% pageurl page.get_parent %}?case_statuses={{ case_statuses }}">
+				{% get_case_status_display case_status as case_status_display_name %}
+				{{ case_status_display_name|capfirst }}
+			</a>
+			{% if not forloop.last %}, {% endif %}
+			{% endfor %}
```

Comment:
If an incident has more than one status, I'm noticing some extra spacing around the comma in the list:

![image](https://user-images.githubusercontent.com/561931/143940765-9529f0f6-540f-4349-935b-a4a3b10d984c.png)

There are probably multiple ways to address this but I was able to make it go away by removing the whitespace in the template like this:

```
{{ case_status_display_name|capfirst }}</a>{% if not forloop.last %}, {% endif %}
```

Potentially the [spaceless template tag](https://docs.djangoproject.com/en/3.2/ref/templates/builtins/#spaceless) could be used as well.